massa-node/base_config/config.toml

```
@@ -89,10 +91,14 @@
     max_simultaneous_ask_blocks_per_node = 128
     # max milliseconds to wait while sending an event before dropping it
     max_send_wait = 0
+    # max cache size for which operations your node knows about
+    max_known_ops_size = 50000
```

Let's go even further: 
```suggestion
    max_known_ops_size = 200000
```